sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingDeduplicationSuite.scala

```diff
@@ -31,8 +31,11 @@ class StreamingDeduplicationSuite extends StateStoreMetricsTest with BeforeAndAf
   import testImplicits._
 
```

Comment:
Here, too. Let's remove the redundant `with BeforeAndAfterAll` and the whole `afterAll` function.